sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -454,24 +454,24 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with BeforeAndAfterE
   }
 
   test("SPARK-30049 Should not complain for quotes in commented lines") {
-    runCliWithin(1.minute)(
+    runCliWithin(3.minute)(
```

Comment:
Why did we need this change?

Reply:
I got timeouts running in a slow machine, changing back to 1.